ci/workflows/pull-request.yml

```diff
@@ -198,7 +198,7 @@ steps:
           run: source-test-env
           upload-container-logs: always
       - ./ci/plugins/upload-failure-logs
-    timeout_in_minutes: 20
+    timeout_in_minutes: 25
```

Comment:
Can you trigger main-cron to run also for this workflow. The timeout for main-cron may need to be adjusted as well.